Makefile

```diff
@@ -271,7 +271,7 @@ ifeq (,$(shell which operator-sdk 2>/dev/null))
 	set -e ;\
 	mkdir -p $(dir $(OPERATOR_SDK)) ;\
 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.14.0/operator-sdk_$${OS}_$${ARCH};\
+	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v1.19.1/operator-sdk_$${OS}_$${ARCH};\
```

Comment:
just curious, no side effects in bumping sdk version right?

Reply:
honestly, I don't think there should be problems as we are not updating to a major version.